docs/src/examples/components/Menu/Usages/MenuExampleToolbar.shorthand.steps.ts

```diff
@@ -0,0 +1,40 @@
+import { Menu } from '@stardust-ui/react'
+
+interface StepsOptions {
+  vertical?: boolean
+}
+
+const selectors = {
+  menu: `.${Menu.className}`,
+  item: (itemIndex: number) => `.${Menu.className} li:nth-child(${itemIndex}) a`,
+  lastItem: `.${Menu.className} li:last-child a`,
+}
+
+const getScreenerSteps = ({ vertical }: StepsOptions = {}): ScreenerSteps => [
```

Comment:
first 4 snapshots in this method are the same as in `getScreenerSteps` from  `docs/src/examples/components/Menu/commonScreenerSteps.ts`; let's reuse that and do:

```typescript
import { Menu } from '@stardust-ui/react'

import getScreenerSteps from '../commonScreenerSteps'

const selectors = {
  menu: `.${Menu.className}`,
  item: (itemIndex: number) => `.${Menu.className} li:nth-child(${itemIndex}) a`,
  lastItem: `.${Menu.className} li:last-child a`,
}

const config: ScreenerTestsConfig = {
  steps: [
    (builder, keys) =>
      getScreenerSteps()[0](builder, keys)
        .click(selectors.lastItem)
        .snapshot('Clicks on the last item and opens submenu')

        .keys(selectors.lastItem, keys.downArrow)
        .snapshot('Focuses on the first element in the submenu'),
  ],
}

export default config
```

I will think of some way to make this more readable, but better to reuse what we have